tophat/jenkins-stats-publisher-plugin
# feat: send run duration and result to datadog

linking fetching jenkins run to sending to data dog

DevQa 
1. create a folder and inside the folder a pipeline in jenkins
2. use SampleJenkinsFile for pipeline and build the pipeline
3. after seeing log sent in jenkin's system logs go to Marc's DataDog dashboard
4. add new timeseries graph for duration and query on `<FOLDER_NAME>.<PIPELINE_NAME>.duration` (the folder and pipeline name from jenkins)
5. add new timeseries graph for result and query on `<FOLDER_NAME>.<PIPELINE_NAME>.result`

Duration graph should have value of how long the build took
Result graph should only have the value 1 if a failure occurred
